TerminalDocs/terminal-parameters.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Windows Terminal Command Line Parameters
+description: Command line parameters in the Windows Terminal.
+author: cinnamon-msft
+ms.author: cinnamon
+ms.date: 05/19/2020
+ms.topic: include
+ms.service: terminal
+---
+| `--profile,-p profile-name` | Use the given profile to open the new tab/pane, where `profile-name` is the `name` or `guid` of a profile. If `profile-name` does not match _any_ profiles, uses the default. |
+| `--startingDirectory,-d starting-directory` | Overrides the value of `startingDirectory` of the specified profile, to start in `starting-directory` instead. |
+| `commandline` | A command line to replace the default command line of the selected profile. If the user wants to use a `;` in this command line, it should be escaped as `\;`. |
```

Comment:
I think this table doesn't render properly (I put another comment on this somewhere). Idk why but I wanted to point it out.